packages/styles/components/_c.toggle.scss

```diff
@@ -103,6 +106,7 @@
           "notification-available-16",
           $color-toggle-disabled-checked-background
         ));
+        background-size: contain;
```

Comment:
~~background-size: contain;~~ => Due to the "cascade", it is not necessary to repeat this value here.